src/promptflow/promptflow/exceptions.py

```diff
@@ -312,7 +312,6 @@ def _module_target_map(cls) -> Dict[str, ErrorTarget]:
             "promptflow._core": ErrorTarget.EXECUTOR,
             "promptflow.batch": ErrorTarget.EXECUTOR,
             "promptflow.contracts": ErrorTarget.EXECUTOR,
-            "promptflow._utils": ErrorTarget.EXECUTOR,
```

Comment:
I think the error target for utils should be removed as executor, for example, in the following error, the target should be sdk instead of executor:

promptflow._sdk._telemetry.activity, line 155, yield logger
promptflow._cli._pf._connection, line 254, create_connection(args.file, args.params_override, args.name)
promptflow._cli._utils, line 372, raise e
promptflow._cli._utils, line 360, return func(*args, **kwargs)
promptflow._cli._pf._connection, line 193, connection = load_connection(source=file_path, params_override=params_override)
promptflow._sdk._load_functions, line 120, return load_common(_Connection, source, **kwargs)
**promptflow._sdk._load_functions, line 51, yaml_dict = load_yaml(source)
promptflow._utils.yaml_utils, line 62, raise Exception(msg.format(source))**

Reply:
But most of the _utils are not maintained by sdk crew : 
![image](https://github.com/microsoft/promptflow/assets/47586720/4fcd8615-66df-4d52-978d-ad3cb9df3267)

Reply:
what's the influence of this deletion, will error in this module has no error target anymore?

Reply:
I have already communicated with Teams and classified them based on the upstream packages of utils. If they cannot be found, the default is executor.